Justfile

```diff
@@ -3,39 +3,45 @@ TS_LOADER := " --import @swc-node/register/esm-register"
 NODE_PARAMS := "--no-warnings " + TS_LOADER
 
 _default:
-  @just --list
+    @just --list
 
 server *node_args="":
-  @node {{NODE_PARAMS}} {{node_args}} packages/server/main.ts --start
+    @node {{ NODE_PARAMS }} {{ node_args }} packages/server/main.ts --start
 
 linter:
-  @npx @biomejs/biome check
+    @npx @biomejs/biome check
 
 test *args="":
-  @npx vitest run {{args}}
+    @npx vitest run {{ args }}
 
 test-watch *args="":
-  @npx vitest watch {{args}}
+    @npx vitest watch {{ args }}
 
 build-console:
-  just _pre-build
-  @rm -rf build/console
-  @cd packages/console && npm run build
+    just _pre-build
+    @rm -rf build/console
+    @cd packages/console && npm run build
+
+dev-console:
+    just _pre-build
+    @rm -rf build/console
+    @cd packages/console && npm run build-watch
```

Comment:
Added `dev-console` command to improve experience when developing on the `console` package